src/hooks/useSnackbar/useSnackbar.tsx

```diff
@@ -10,13 +10,16 @@ import { createId } from '@/utils/createId'
 type SnackbarIconType = 'success' | 'error' | 'info' | 'warning'
 
 export type DisplaySnackbarArgs = {
+  customId?: string
   timeout?: number
   variant?: 'primary' | 'secondary'
   iconType?: SnackbarIconType
   title: string
   description?: string
   actionText?: string
+  onExit?: () => void
   onActionClick?: () => void
+  onExitClick?: () => void
```

Comment:
I see why we need `onExit` now, I guess one question I have is do we need separate callback for exit and exit click? Maybe both of those could be just `onExit`?